src/kubeobjects/env_test.go

```diff
@@ -0,0 +1,39 @@
+package kubeobjects
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+)
+
+func TestFindEnvVar(t *testing.T) {
+	envVars := []v1.EnvVar{
+		{Name: testKey1, Value: testAppVersion},
```

Comment:
Nit: `testKey1` is defined in `secrets_tests.go`. It would be convenient to create `consts.go` to share const values between different tests.

Reply:
Moved the constants, I named the file `consts_test.go` so they are only compiled and available when running unit tests